source/includes/reference/_borderless.md

```diff
@@ -348,7 +430,7 @@ query.accountId                | Query parameter repeated         | Integer
 > Example Request:
 
 ```shell
-curl -X POST https://api.sandbox.transferwise.tech/v1/borderless-accounts/{borderlessAccountId}/conversions \
+curl -X POST https://api.sandbox.transferwise.tech/v2/borderless-accounts/{borderlessAccountId}/conversions \
```

Comment:
The interesting bit here is that once you know the id of the account you do not need to use your profile id to validate this.
However, in the multi user access world how do you plan on making sure you allow the right users to execute conversion?
I guess that for start you plan on allowing only the owner of the profile to convert, but then is my accountant allowed to do it?

I guess one easy solution is to use:
```
curl -X POST https://api.sandbox.transferwise.tech/v2/profiles/{id}/borderless-accounts/{borderlessAccountId}/conversions
```
But the ugliness of this is that we are requiring our api customers to know the right mapping of profile -> balance in order to do conversions, which is in effect not their problem, but ours.
Knowing the user and the balance we will be able to infer if the customer is allowed to convert or not, but that means that balance needs to perform authorization.

I am curious what are your thoughts about the evolution on this.

Reply:
Yeah, I'm not sure I have the right answer here. There clearly is a mapping of profile -> borderless-account but no easy way to solve this mapping without introducing some permission similar to the profile one for this resource. Otherwise you always would have to query balance to get the profile behind it.

In terms of resource ownership the full path does make sense, as the account(s) is(are) owned by a certain profile so I'd tend to prefer this as it looks more restful 
```
/profiles/{profile_id}/borderless-accounts/{account_id}
```